resources/js/Shared/Layout/AdminNavigation.vue

```diff
@@ -18,48 +29,43 @@ const props = defineProps({
       </Link>
 
       <div class="mr-3.5 flex sm:hidden">
-        <button type="button" class="inline-flex items-center justify-center rounded-md p-2.5 text-gray-700" @click="mobileMenuOpen = true">
+        <button type="button" class="inline-flex items-center justify-center rounded-md p-2.5 text-gray-700" @click="toggleMobileMenu">
           <span class="sr-only">Open main menu</span>
           <Bars3Icon class="h-6 w-6" aria-hidden="true" />
         </button>
       </div>
+      <Dialog v-if="isMobileMenuOpened" as="div" class="z-30 lg:hidden" :open="isMobileMenuOpened" @close="toggleMobileMenu">
+        <div class="fixed inset-0 z-30" />
+        <DialogPanel class="fixed inset-y-0 right-0 z-30 w-full overflow-y-auto border-b-2 bg-white px-6 py-3 sm:max-w-sm sm:ring-1 sm:ring-gray-900/10">
+          <div class="flex items-center justify-between sm:justify-end">
+            <Link href="/">
+              <img class="h-10 sm:hidden" src="@/assets/scooter.png" alt="escooter logo">
+            </Link>
+            <button type="button" class="-m-2.5 rounded-md px-2.5 text-gray-700 sm:pt-4" @click="toggleMobileMenu">
+              <span class="sr-only">Close menu</span>
+              <XMarkIcon class="h-6 w-6" aria-hidden="true" />
+            </button>
+          </div>
+          <div class="mt-6 flow-root">
+            <div class="-my-6 divide-y divide-gray-500/10">
+              <div class="space-y-2 py-6">
+                <Link v-for="item in navigation" :key="item.name" :class="$page.url.startsWith(item.href)? 'bg-blumilk-50' : ''" :href="item.href" class="-mx-3 block rounded-lg px-3 py-2 text-base font-semibold leading-7 text-gray-800 hover:bg-blumilk-25" @click="clearFilters">
```

Comment:
```suggestion
                <Link v-for="item in navigation" :key="item.name" :class="{'bg-blumilk-50': $page.url.startsWith(item.href)}" :href="item.href" class="-mx-3 block rounded-lg px-3 py-2 text-base font-semibold leading-7 text-gray-800 hover:bg-blumilk-25" @click="clearFilters">
```
`:class` can be simplified

Reply:
changed